ODIN_II/regression_test/benchmark/task/keywords/@()/task.conf

```diff
@@ -0,0 +1,23 @@
+########################
+# operators benchmarks config
+########################
+
+regression_params=--include_default_arch
+script_synthesis_params=--time_limit 3600s 
+script_simulation_params=--time_limit 3600s
+simulation_params= -L reset rst -H we
+
+# setup the architecture
+archs_dir=../vtr_flow/arch/timing
+
+arch_list_add=k6_N10_40nm.xml
+arch_list_add=k6_N10_mem32K_40nm.xml
+arch_list_add=k6_frac_N10_frac_chain_mem32K_40nm.xml
+
+# setup the circuits
+circuits_dir=regression_test/benchmark/verilog/keywords
+
+circuit_list_add=@()/*.v
```

Comment:
does this actually work? I'm worried bash might not like the round bracket unles they are escaped.